include/qpdf/QPDF.hh

```diff
@@ -399,6 +399,11 @@ class QPDF
     QPDFObjectHandle getObjectByObjGen(QPDFObjGen const&);
     QPDF_DLL
     QPDFObjectHandle getObjectByID(int objid, int generation);
+    // The following overload is for internal use and should not
+    // be required by users of the library. Setting resolve to false will
+    // stop getObject from trying to resolve an unresolved object.
+    QPDF_DLL
+    QPDFObjectHandle getObject(QPDFObjGen const&, bool resolve);
```

Comment:
If this is really only for `QPDFParser` to call, I'd rather see a private method called something like `getUnresolvedObject` and have it protected by one of those little wrapper classes I like to make that has `QPDFParser` as a friend. You know, something like `QPDF::GetUnresolved::getUnresolved(context, ref_og)`. See summary comments.